src/main/java/org/example/daos/ProjectDao.java

```diff
@@ -0,0 +1,168 @@
+package org.example.daos;
+
+import org.example.models.DeliveryEmployee;
+import org.example.models.Project;
+import org.example.models.ProjectRequest;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProjectDao {
+    private enum Indexes {
+        name(1),
+        value(2),
+        techLeadId(3),
+        clientId(4),
+        salesEmployeeId(5),
+        startDate(6),
+        finishDate(7),
+        updateId(8);
```

Comment:
enums should be formatted in upper case as they are effectively private static variables